GNUmakefile

```diff
@@ -82,7 +82,7 @@ abigen: ## Build & install abigen.
 	./tools/bin/build_abigen
 
 .PHONY: generate
-generate: abigen codecgen mockery ## Execute all go:generate commands.
+generate: mockery ## Execute all go:generate commands.
```

Comment:
I'll revert this on the next revision - I remove abigen and codecgen since on my local machine they make updates